pom.xml

```diff
@@ -389,6 +389,24 @@
                 <show>protected</show>
                 <nohelp>true</nohelp>
                 <outputDirectory>${project.build.directory}/javadoc</outputDirectory>
+                <groups>
+                  <group>
+                    <title>Main packages</title>
+                    <packages>com.google.gcloud*</packages>
+                  </group>
+                  <group>
+                    <title>Test helpers packages</title>
+                    <packages>com.google.gcloud.bigquery.testing:com.google.gcloud.datastore.testing:com.google.gcloud.resourcemanager.testing:com.google.gcloud.storage.testing</packages>
+                  </group>
+                  <group>
+                    <title>SPI packages</title>
+                    <packages>com.google.gcloud.spi</packages>
```

Comment:
Looks really great...
Maybe a different ordering (i.e API first, then SPI then Test, then
Examples)?
Also since we mention "SPI packages", should we rename the "Main packages"
as "API packages"?


On Thu, Mar 10, 2016 at 10:12 AM, Arie <notifications@github.com> wrote:

> In pom.xml
> <https://github.com/GoogleCloudPlatform/gcloud-java/pull/738#discussion_r55722954>
> :
>
> > @@ -389,6 +389,24 @@
> >                  <show>protected</show>
> >                  <nohelp>true</nohelp>
> >                  <outputDirectory>${project.build.directory}/javadoc</outputDirectory>
> > +                <groups>
> > +                  <group>
> > +                    <title>Main packages</title>
> > +                    <packages>com.google.gcloud*</packages>
> > +                  </group>
> > +                  <group>
> > +                    <title>Test helpers packages</title>
> > +                    <packages>com.google.gcloud.bigquery.testing:com.google.gcloud.datastore.testing:com.google.gcloud.resourcemanager.testing:com.google.gcloud.storage.testing</packages>
> > +                  </group>
> > +                  <group>
> > +                    <title>SPI packages</title>
> > +                    <packages>com.google.gcloud.spi</packages>
>
> This brings me back to #422
> <https://github.com/GoogleCloudPlatform/gcloud-java/issues/422>. Any
> thoughts?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/gcloud-java/pull/738/files#r55722954>
> .
>